braindecode/classifier.py

```diff
@@ -33,15 +33,21 @@ class name. Name conflicts are resolved by appending a count suffix
         callbacks (e.g., for the callback with name ``'print_log'``, use
         ``net.set_params(callbacks__print_log__keys_ignored=['epoch',
         'train_loss'])``).
+    iterator_train__shuffle: bool (default=True)
+        Defines whether train dataset will be shuffled. As skorch does not
+        shuffle the train dataset by default this one overwrites this option.
     """
     # TODO: Update docstring to use NeuralNetClassifier docstring with some
     #  imporvements
 
     def __init__(self, *args, cropped=False, callbacks=None, **kwargs):
         self.cropped = cropped
         callbacks = self._parse_callbacks(callbacks)
-
-        super().__init__(*args, callbacks=callbacks, **kwargs)
+        iterator_train__shuffle = kwargs.pop('iterator_train__shuffle', True)
```

Comment:
why not just put it into __init__ function signature? then also shows up in IDE etc. and then no need to do this kwargs.pop stuffs